spec/models/site_setting_spec.rb

```diff
@@ -69,6 +69,40 @@
     end
   end
 
+  describe "min_redirected_to_top_period" do
+
+    context "has_enough_top_topics" do
+
+      SiteSetting.topics_per_period_in_top_page = 2
+      SiteSetting.top_page_default_timeframe = 'daily'
+
+      before do
+        SiteSetting.expects(:min_redirected_to_top_period).returns(:daily)
+      end
+
+      it "should_return_a_time_period" do
+        expect(SiteSetting.min_redirected_to_top_period(1.days.ago)).not_to eq(nil)
```

Comment:
I think this is only testing that the stubbing works properly ;) If you comment out the content of the `min_redirected_to_top_period` method, are the tests failing?